index.html

```diff
@@ -1,60 +1,113 @@
+        <BR>
         </td>
       </tr>
-  		<tr class="spaceUnder" align="center">
-  			<td>
-          <button type="button" class="btn btn-success" id="save_now" tabindex='-1'>Save Page Now</button>
-          <BR>
-    		</td>
-      </tr>
-      <tr class="spaceUnder" align="center">
-    		<td>
-          <button type="button" class="btn btn-primary" id="recent_capture" tabindex='-1'>Recent Version</button>
-          <BR>
-    		</td>
-      </tr>
       <tr class="spaceUnder" align="center">
         <td>
-          <button type="button" class="btn btn-warning" id="first_capture" tabindex='-1'>First Version</button>
-        </td>
-    	</tr>
-    </table>
-    <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
-  </body>
-</html>
-<script src="scripts/jquery.js"></script>
-<script src="scripts/popup.js"></script>
-<script src="scripts/bootstrap.min.js"></script>
-<style>
-  tr.spaceUnder > td
-  {
-    padding-bottom: 1em;
-  }
-  .container {
-    margin: 2px;
-    width: auto;
-    height: 45px;
-    line-height: 45px;
-    text-align: center;
-  }
-  .resize_fit_center {
-      max-width:100%;
-      max-height:100%;
-      vertical-align: middle;
-  }
-</style>
\ No newline at end of file
+        <div >            
+            <a id="facebook" class="btn btn-social-icon btn-sm btn-facebook btn-indigo"><i class="fa fa-facebook"></i></a>    
+            <a id="google" class="btn btn-social-icon btn-sm btn-google-plus btn-red"><i class="fa fa-google-plus"></i></a>
+            <a id="linkedin" class="btn btn-social-icon btn-sm btn-linkedin btn-light-blue"><i class="fa fa-linkedin"></i></a>
+            <a id="twitter" class="btn btn-social-icon btn-sm btn-twitter btn-blue"><i class="fa fa-twitter"></i></a>
+          </div>
+
+          <BR>
+          </td>
+        </tr>
+      </table>
+      <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
+    </body>
+    </html>
+    <script src="scripts/popup.js"></script>
+    <script src="scripts/bootstrap.min.js"></script>
```

Comment:
@rgpt thanks for taking the time to go through my code and reviewing it. As for your review, I have already seen the error and it is a custom exception which is thrown if jquery is not there. Even if it is being shown on the console, it doesn't have any effect on speed or efficiency of extension.